Autopilot/Test/Src/PathManager/Test_ModeSelector_Queue.cpp

```diff
@@ -0,0 +1,201 @@
+/*
```

Comment:
Could you give a rundown of which tests you're running here (in the comments or in documentation) for future reference?